src/core/DataFile.cpp

```diff
@@ -898,6 +899,25 @@ void DataFile::upgrade_1_1_91()
 			el.setAttribute( "arpdir", "3" );
 		}
 	}
+
```

Comment:
This should be in it's own upgrade function, right?
 void DataFile::upgrade_1_1_91()
What's the new upgrade function called `_1_1_92()` ? `_1_2_0_rc2` ?